admin/app/helpers/spree/admin/currencies_helper.rb

```diff
@@ -2,7 +2,7 @@ module Spree
   module Admin
     module CurrenciesHelper
       def preferred_currencies
-        @preferred_currencies ||= ([current_store.default_currency] + current_store.supported_currencies_list).uniq
+        @preferred_currencies ||= current_store.supported_currencies_list
```

Comment:
1. `supported_currencies_list` already includes `default_currency`
2. `default_currency` is a String and `supported_currencies_list` is an Array of `Money::Currency`